src/components/search-bar/search-bar.component.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Dropdown from 'react-bootstrap/Dropdown';
+import Button from 'react-bootstrap/Button';
+
+import './search-bar.style.scss';
+
+/**
+ * Search bar
+ */
+export default class SearchBar extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            userInput: '',
+            searchType: 'email'
+        };
+
+        this.updateSearchValue = this.updateSearchValue.bind(this);
+        this.searchClicked = this.searchClicked.bind(this);
+        this.searchTypeSelect = this.searchTypeSelect.bind(this);
+    }
+
+    searchTypeSelect(key) {
+        this.setState({ searchType: key });
+    }
+
+    updateSearchValue(e) {
+        this.setState({ userInput: e.target.value });
+    }
+
+    searchClicked() {
+        this.onSubmit(this.state.userInput, this.state.searchType);
```

Comment:
this.props.onSubmit?
```suggestion
        this.props.onSubmit(this.state.userInput, this.state.searchType);
```